nextflow.config

```diff
@@ -64,6 +64,8 @@ if ( params.illumina ){
 
 executor{
      queueSize = params.myQueueSize
+     submitRateLimit = '5sec'
+     pollInterval = '15 sec'
```

Comment:
I'm still tweaking the config so we can execute the pipeline efficiently. Used these setting and completed one 2000 sample batch successfully. I am not 100% confident these settings are bullet proof though.